src/lib/data/createables.ts

```diff
@@ -366,6 +366,56 @@ const hunterClothing: Createable[] = [
 ];
 
 const Reverteables: Createable[] = [
+	{
+		name: 'Revert tanzanite fang',
+		inputItems: {
+			[itemID('tanzanite fang')]: 1
+		},
+		outputItems: {
+			[itemID("Zulrah's scales")]: 20_000
+		},
+		noCl: true
+	},
+	{
+		name: 'Revert toxic blowpipe (empty)',
+		inputItems: {
+			[itemID('tanzanite fang')]: 1
+		},
+		outputItems: {
+			[itemID("Zulrah's scales")]: 20_000
+		},
+		noCl: true
+	},
+	{
+		name: 'Revert magic fang',
+		inputItems: {
+			[itemID('magic fang')]: 1
+		},
+		outputItems: {
+			[itemID("Zulrah's scales")]: 20_000
+		},
+		noCl: true
+	},
+	{
+		name: 'Revert serpentine visage',
+		inputItems: {
+			[itemID('serpentine visage')]: 1
+		},
+		outputItems: {
+			[itemID("Zulrah's scales")]: 20_000
+		},
+		noCl: true
+	},
+	{
+		name: 'Revert serpentine helm (uncharged)',
+		inputItems: {
+			[itemID('serpentine helm (uncharged)')]: 1
```

Comment:
these need to be cased properly (also elsewhere)

Reply:
Correctly capitalized all the zulrah reverables now.